openhtf/plugs/user_input.py

```diff
@@ -38,7 +37,7 @@
 from six.moves import input
 
 if platform.system() != 'Windows':
-  import termios
+  import termios  # pylint: disable=g-import-not-at-top
```

Comment:
I am not sure this comments belongs in the github source

Reply:
It's common in protobuf code and has been used in other OpenHTF files.